stdlib/public/SDK/SceneKit/SceneKit.swift

```diff
@@ -151,7 +151,7 @@ extension SCNGeometryElement {
   /// Creates an instance from `indices` for a `primitiveType`
   /// that has a constant number of indices per primitive
   /// - Precondition: the `primitiveType` must be `.triangles`, `.triangleStrip`, `.line` or `.point`
-  public convenience init<IndexType : Integer>(
+  public convenience init<IndexType : FixedWidthInteger>(
```

Comment:
Oh!  Heck, their constraint is wrong to begin with.  They are assuming all `Integer`s are POD.  We should file a radar.